eve/main.yml

```diff
@@ -280,9 +280,9 @@ stages:
       - ShellCommand:
           name: Build docker image
           command: >-
-            docker build
-            -t %(prop:docker_image_name)s
-            -t zenko/cloudserver:latest-%(prop:product_version)s
+            docker build --no-cache
+            -t "%(prop:docker_image_name)s"
+            -t "zenko\/cloudserver:latest-%(prop:product_version)s"
```

Comment:
wow if we reached to this point, maybe it will be better to put those commands on a script/makefile?